### PR TITLE
Fsrv 1308

### PR DESCRIPTION
**Pull request by faheemziker**
1) requirements file changed to use prod version of injector
2) settings updated (for qa server)
3) settings updated (login url and institution info urls changed)
4) users file updated to authenticate using admin.artstor.org

### CODE REVIEW

File: users.py [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
+    if j['status'] in [0, '0']:
+        return None
+    try:
+        headers = {'Cookie': response.headers['Set-Cookie']}
+        institution_url = settings.getConfig("institution_url")
+        response = requests.request("GET", institution_url, headers=headers)
+        j = response.json()
+
+        i_id = j['institution']['id']
+        name = j['institution']['display_name']
```
Comment (cjzeng) [inline users.py]: Should use "name" instead of "display_name" as the response does not contain that value.

Comment (faheemziker) [reply users.py]: change request done

**Comment by cjzeng** [comment]
Name changes code looks good.
Still unable to run it locally to test out the code based on README

**Comment by faheemziker** [comment]
can you explain what is the issue on your side while trying it locally..?

**Pull request merged**